### PR TITLE
langjam entry from guitarvydas

### PR DESCRIPTION
**Pull request by guitarvydas**
see README.md

### CODE REVIEW

File: component.pl [hunk 1]
```diff
@@ -0,0 +1,74 @@
+component(Diagram,C,Name,Ins,Outs,Children,Connections) :-
```
Comment (jntrnr) [inline component.pl:1]: All the files for your entry need to be in your team's directory

Comment (guitarvydas) [reply component.pl:1]: I haven't done this before and I haven't done a forked PR before.

I need clarification, if you have time...

I have not included node_modules and /usr/local/bin/swipl.  Swipl is a PROLOG compiler (SWI Prolog).

Should I be including these in the PR?

It is not obvious to me what is missing.

thanks!
pt

Comment (AmrDeveloper) [reply component.pl:1]: Hello guitarvydas,

In your computer move all your project files from `jam0001` to your `jam0001/guitarvydas` directory

for example, your files `component.pl` and `jsoncomponent.pl` are on the `jam0001` directory outside your team directory

Just move them and then create and push a new commit, no need to create another PR

help

Comment (AmrDeveloper) [reply component.pl]: @guitarvydas Your `.gitignore` also should be in your project directory because it is related to it, not to all project

**Comment by guitarvydas** [comment]
Ah, thank you!

Those two files (plus my .gitignore) are already in the guitarvydas directory and old copies somehow leaked into the upper directory.  I've deleted the 3 offending files from the main directory and have re-pushed.

**Comment by guitarvydas** [comment]
Let me know if the re-push has worked.

**Comment by AmrDeveloper** [comment]
@guitarvydas It's looking good now 
Wait for @jntrnr review

**Comment by jntrnr** [comment]
Confirmed that only files were removed after cut-off. Thanks for cleaning this up. Should be good.

**Pull request merged**